proto/interchain_security/ccv/v1/ccv.proto

```diff
@@ -30,3 +32,28 @@ enum Status {
   // channel is invalid and can no longer process packets
   STATUS_INVALID = 3 [(gogoproto.enumvalue_customname) = "INVALID"];
 }
+
+// CreateChildChainProposal is a governance proposal on the parent chain to spawn a new child chain.
+// If it passes, then all validators on the parent chain are expected to validate the child chain at spawn time
+// or get slashed. It is recommended that spawn time occurs after the proposal end time.
+message CreateChildChainProposal {
+  option (gogoproto.goproto_getters)  = false;
+  option (gogoproto.goproto_stringer) = false;
+
+  // the title of the proposal
+  string title = 1;
+  // the description of the proposal
+  string description = 2;
+  // the proposed chain-id of the new child chain, must be different from all other child chain ids of the executing
+  // parent chain.
+  string chain_id = 3 [(gogoproto.moretags) = "yaml:\"chain_id\""];
+  // the proposed client state of new child chain.
+  // Since it contains no consensus information, this may be created before child chain is created.
+  google.protobuf.Any client_state = 4 [(gogoproto.moretags) = "yaml:\"client_state\""];
+  // genesis hash with no staking information included.
```

Comment:
> with no staking information included

i.e without the genesis module included? We should also specify in the comment that this is the child chain's genesis hash